### PR TITLE
Session doc changes

### PR DESCRIPTION
**Pull request by austingmhuang**

### CODE REVIEW

**Review by mudit2812 — COMMENTED** [review]
Can you also add a changelog entry 🙏

**Review by mudit2812 — APPROVED** [review]

File: pennylane_qiskit/qiskit_device.py [hunk 1]
```diff
@@ -84,20 +84,28 @@ def execute(self, circuits, execution_config=DefaultExecutionConfig):
...
-    on the device while the context manager is active. Using the context manager
-    will ensure the Session closes properly and is removed from the device after
-    completing the tasks. Any Session that was initialized and passed into the
+    """
+    A context manager that creates a Qiskit Session and sets it as a session
+    on the device while the context manager is active.
+
+    .. warning::
+
+        At the time of writing, sessions can no longer be used by IBM users on
```
Comment (soranjh) [inline pennylane_qiskit/qiskit_device.py:93]:
```suggestion
        Currently, sessions cannot be used by IBM users on
```

File: pennylane_qiskit/qiskit_device.py [hunk 2]
```diff
@@ -84,20 +84,28 @@ def execute(self, circuits, execution_config=DefaultExecutionConfig):
...
+        the Open plan. There are also reports of sessions behaving unexpectedly and closing randomly.
+        In such an event, we recommend referring to the Qiskit Session
+        `documentation <https://docs.quantum.ibm.com/api/qiskit-ibm-runtime/qiskit_ibm_runtime.Session>`_
+        and opening a session with Qiskit's session directly while we work to resolve these issues.
```
Comment (soranjh) [inline pennylane_qiskit/qiskit_device.py:97]:
```suggestion
        the Open plan. We recommend referring to the Qiskit Session
        `documentation <https://docs.quantum.ibm.com/api/qiskit-ibm-runtime/qiskit_ibm_runtime.Session>`_
        and opening a session with Qiskit's session directly.
```

File: pennylane_qiskit/qiskit_device.py [hunk 3]
```diff
@@ -84,20 +84,28 @@ def execute(self, circuits, execution_config=DefaultExecutionConfig):
...
     Args:
         device (QiskitDevice2): the device that will create remote tasks using the session
-        **kwargs: session keyword arguments to be used for settings for the Session. At the
-            time of writing, the only relevant keyword argument is "max_time", which lets you
-            set the maximum amount of time the session is open. For the most up to date information,
-            please refer to the Qiskit Session
-            `documentation <https://docs.quantum.ibm.com/api/qiskit-ibm-runtime/qiskit_ibm_runtime.Session>`_.
+        **kwargs: keyword arguments for session settings. At the time of writing, the only relevant
+            keyword argument is "max_time", which lets you set the maximum amount of time the session
+            is open.
```
Comment (soranjh) [inline pennylane_qiskit/qiskit_device.py:108]:
```suggestion
        **kwargs: keyword arguments for session settings. Currently, the only relevant
            keyword argument is "max_time", which allows setting the maximum amount of time the session
            is open.
```

**Review by soranjh — COMMENTED** [review]
Thanks @austingmhuang, added some suggestions to make the the message clear and brief.

**Review by soranjh — APPROVED** [review]

**Pull request merged**